docs/pages/desktop-access/getting-started.mdx

```diff
@@ -314,6 +314,61 @@ Computer Configuration > Policies > Administrative Templates > Windows Component
   ![Disable Require](../../img/desktop-access/disable.png)
 </Figure>
 
+### Configure certificate for RDP connections
+
+<Admonition type="note" title="Secure Cipher Suites">
+  Teleport's RDP client supports only secure algorithms
+  for making TLS connections, so we have to configure a Windows server
+  to support those cipher suites as well.
+  This step is only *neccessary* for Windows Server 2012 R2 as it does not support
+  secure algorithms by default.
+</Admonition>
+
+
+In this step we'll create a new certificate template that uses
+elliptic curve cryptography, and then configure our GPO
+to use the newly created template to issue certificates used for Remote Desktop connections.
+
+# Create a certificate template
+
+1. Begin by navigating to:
+```
+Start > Control Panel > Administrative Tools > Certificate Authority
+```
+to open the CA Microsoft Management Console (MMC) GUI.
+2. Open your CA computer and right-click on `Certificate Templates`, then select `Manage`.
+3. Find `Computer` template on the list, right-click on it, then select `Duplicate Template`.
+4. In the `Compatibility` tab change `Certification Authority` to `Windows Server 2012 R2` and click `OK`.
+In the same tab change `Certificate recipient` to `Windows Server 2012 R2` and click `OK`.
```

Comment:
Indent items inside the list to avoid breaking it. I would also space the contents, keeping the lines together breaks some markdown parsers.

```suggestion
4. In the `Compatibility` tab change `Certification Authority` to `Windows Server 2012 R2` and click `OK`.

    In the same tab change `Certificate recipient` to `Windows Server 2012 R2` and click `OK`.

```